obspy/core/stream.py

```diff
@@ -2181,7 +2181,7 @@ def simulate(self, paz_remove=None, paz_simulate=None,
         return self
 
     @raise_if_masked
-    def filter(self, type, **options):
+    def filter(self, type, *freqs, **options):
```

Comment:
I would just go with `*args` (yes and in hindsight it should've just been `**kwargs**``and not `**options` but we're stuck with this now)

Reply:
I can change that, but `*freqs` makes it very clear, that only frequencies are expected.